data/xml/1962.earlymt.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1962.earlymt">
+  <volume id="1" ingest-date="2021-10-30">
+    <meta>
+      <booktitle>Automatic translation of languages NATO Summer School</booktitle>
+      <publisher>Pergamon Press</publisher>
+      <address>Venice</address>
+      <month>July</month>
+      <year>1962</year>
+    </meta>
+    <frontmatter>
+      <url hash="6e5c2d0d">1962.earlymt-1.0</url>
+      <bibkey>earlymt-1962-automatic</bibkey>
+    </frontmatter>
+    <paper id="1">
+      <title>Four Lectures on Algebraic Linguistics and Machine Translation</title>
+      <author><first>Yehoshua</first><last>Bar-Hillel</last></author>
+      <pages>1-26</pages>
+      <url hash="16df1606">1962.earlymt-1.1</url>
+      <bibkey>bar-hillel-1962-four</bibkey>
+    </paper>
+    <paper id="2">
+      <title>Automatic Translation of Languages</title>
+      <author><first>A. F. R.</first><last>Brown</last></author>
+      <pages>27-54</pages>
+      <url hash="1750504b">1962.earlymt-1.2</url>
+      <bibkey>brown-1962-automatic</bibkey>
+    </paper>
+    <paper id="3">
+      <title>Automatic Translation of Languages</title>
```

Comment:
Second paper with this name? Maybe delete this volume and its PDFs, then reingest…

Reply:
Yes, the papers have the same name but different authors.